sensor/Instructions.md

```diff
@@ -51,14 +51,25 @@ make updaterdk
 
 Next, run `make bin/customsensor` to build the executable.
 
-#### 6. Test your module locally
+#### 5. Test your module locally
 
-Follow the instructions to [Add a local module](https://docs.viam.com/registry/configure/#add-a-local-module).
+While testing, we recommend that you add the resources you've built with a remote to your machine because it allows you to recompile your code without having to restart your entire machine:
 
-#### 7. Upload your module to the registry
+1. Edit the [`cmd/module/cmd.go` file](./cmd/module/cmd.go). The code there should configure and run a robot with your modular resource configured.
```

Comment:
I don't understand this instruction. The cmd/module is supposed to be a module. It's not clear at all what it means for the "code to configure and run a robot" (which is normally something humans do via App.)

Reply:
it's supposed to reference the cmd/remote/cmd.go file. Sorry. Yeah - I suppose I need to speak to Eliot why he prefers testing this way.